scripts/lake_sheltering_nlcd.R

```diff
@@ -1,4 +1,45 @@
 
+
+get_nlcd_classes <- function(lake.buffer, filename){
```

Comment:
K, just checking that nothing changed with version of land cover I have for modeling.